data/cncf.yaml

```
@@ -1419,6 +1419,20 @@
       check_sets:
         - community
         - code
+- name: kube-state-metrics
+  display_name: Kube State Metrics
+  description: Add-on agent to generate and expose cluster-level metrics. 
+  category: observability
+  logo_url: https://raw.githubusercontent.com/cncf/artwork/master/projects/kubernetes/icon/color/kubernetes-icon-color.svg
+  devstats_url: https://k8s.devstats.cncf.io/
```

This should take folks directly to the KSM dashboard.

```suggestion
  devstats_url: https://k8s.devstats.cncf.io/d/49/github-stats-by-repository?orgId=1&var-period=h24&var-repos=%22kubernetes%2Fkube-state-metrics%22&var-stat=reviewers
```

---

(I'm assuming here that the dashboard links are not prone to change and stable, feel free to ignore this if that's not the case)